### PR TITLE
LRDOCS-4018 updated Layout template articles to 7.1

### PR DESCRIPTION
**Pull request by mwilliams2014**
cc @jbalsas
cc @izaera

### CODE REVIEW

**Comment by mwilliams2014** [comment]
supersedes https://github.com/sez11a/liferay-docs/pull/2692

**Comment by mwilliams2014** [comment]
Includes LRDOCS-4018 and LRDOCS-4402

File: develop/tutorials/articles/14-themes-and-layout-templates/02-layout-templates/00-layout-templates-intro.markdown [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Layout Templates [](id=layout-templates-intro)
+
+@product@ includes several default layout templates out-of-the-box that you 
+can use to organize content on your pages. If you require a custom 
+configuration, you can create layout templates to cover your specific needs. In 
+this section of tutorials, you'll learn how to develop layout templates for 
+@product@.
```
Comment (sez11a) [inline develop/tutorials/articles/14-themes-and-layout-templates/02-layout-templates/00-layout-templates-intro.markdown]: Hey @mwilliams2014 can you improve this intro so it explains what Layout Templates are and why (i.e., use cases) you'd want to create your own?

Comment (mwilliams2014) [reply develop/tutorials/articles/14-themes-and-layout-templates/02-layout-templates/00-layout-templates-intro.markdown]: Sorry this got buried @sez11a. Yes, I'll update it and push the changes.

**Comment by mwilliams2014** [comment]
Updated @sez11a . Please let me know if this needs more work. Thanks!

**Comment by sez11a** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request merged**